lpeterson52/LiveSite
# Redid navbar in cam.js

This PR replaces the original navbar done in wordpress with a new navbar. This PR includes the following changes:

- Bootstrap imports have been updated to the latest version in all foldmecha modules
- Replaced original wordpress navbar with bootstrap navbar within (only) the cam.js module
- Linted html files of modules
- Deleted optional theme import in html files of modules

To test:

- Run live server extension on cam.html to see new navbar
- Run live server extension on other modules to test if the pages still work with the updated bootstrap imports
